content/docs/start/metrics-parameters-plots.md

```diff
@@ -0,0 +1,257 @@
+into plots file `prc.json`:
+
+```json
+{
+  "prc": [
+    { "precision": 0.021473008227975116, "recall": 1.0, "threshold": 0.0 },
+    ...,
+    { "precision": 1.0, "recall": 0.009345794392523364, "threshold": 0.6 }
+  ]
+}
+```
+
+Similarly, it writes arrays for the
+[roc_curve](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_curve.html)
+into `roc.json` for an additional plot.
+
+> DVC doesn't force you to use any specific file names, or even format or
+> structure of a metrics or plots file - it's pretty much user and case defined.
+> Please refer to `dvc metrics` and `dvc plots` for more details.
+
+Let's save this iteration, so we can compare it later:
+
+```dvc
+$ git add scores.json prc.json roc.json
+$ git commit -a -m "Create evaluation stage"
+```
+
+Later we will see how these and other can be used to compare and visualize
+different pipeline iterations. For now, let's see how can we capture another
+important piece of information that will be useful for comparison: parameters.
+
+## Defining parameters
+
+It's pretty common for data science pipelines to include configuration files
+that define adjustable parameters to train a model, do pre-processing, etc. DVC
+provides a mechanism for stages to depend on the values of specific sections of
+such a config file (YAML, JSON, TOML, and Python formats are supported).
+
+Luckily, we should already have a stage with
+[parameters](/doc/command-reference/params) in `dvc.yaml`:
+
+```yaml
+featurize:
+  cmd: python src/featurization.py data/prepared data/features
+  deps:
+    - data/prepared
+    - src/featurization.py
+  params:
+    - featurize.max_features
+    - featurize.ngrams
+  outs:
+    - data/features
+```
+
+<details>
+
+### 💡 Expand to recall how it was generated.
+
+The `featurize` stage
+[was created](/doc/start/data-pipelines#dependency-graphs-dags) with this
+`dvc run` command. Notice the argument sent to the `-p` option (short for
+`--params`):
+
+```dvc
+$ dvc run -n featurize \
+          -p featurize.max_features,featurize.ngrams \
+          -d src/featurization.py -d data/prepared \
+          -o data/features \
+          python src/featurization.py data/prepared data/features
+```
+
+</details>
+
+The `params` section defines the [parameter](/doc/command-reference/params)
+dependencies of the `featurize` stage. By default DVC reads those values
+(`featurize.max_features` and `featurize.ngrams`) from a `params.yaml` file. But
+as with metrics and plots, parameter file names and structure can also be user
+and case defined.
+
+This is how our `params.yaml` file looks like:
+
+```yaml
+prepare:
+  split: 0.20
+  seed: 20170428
+
+featurize:
+  max_features: 500
+  ngrams: 1
+
+train:
+  seed: 20170428
+  n_est: 50
+  min_split: 2
+```
+
+## Updating and running
+
+Let's edit the `params.yaml` file to use bigrams and increase the number of
```

Comment:
"Updating and running" sounds incomplete. I'd call the previous section **Defining stage parameters** and this one **Tuning params and iterating** or something like that.

Reply:
Also, what was wrong with the previous opening sentence? 🙂 "We are definitely not happy with the AUC value we got so far!"

Reply:
These changes might have been a stretch. I was trying to get away from this sounding like an intro to tuning parameters, since that's more of what's covered in the experiments page. This is also pretty artificial tuning, since in most cases bigrams are going to be better (it's not really experimenting if we know the answer). Maybe we change "tuning" to "updating" and leave the rest as it was? Thoughts?

Reply:
Got it. Alright